### ISSUE TITLE
Generate customize pre-trained

### ISSUE THREAD

**Issue by sigmagod** [junyanz/pytorch-CycleGAN-and-pix2pix#423]
Hi,
  Can I create my own customize pre-trained result for my train result dataset ?

**Comment by junyanz** [comment]
What does "customize pre-trained result“ refer to?

**Comment by sigmagod** [comment]
Sorry, I follow the step from this repo and try to training a module from my datasets, its look like 
![blender_008](https://user-images.githubusercontent.com/21048834/48047204-4a41ad00-e1d2-11e8-9228-216c2e27b4b9.png)  and
![eva](https://user-images.githubusercontent.com/21048834/48047214-5299e800-e1d2-11e8-8479-c4dc3c46a80b.jpg)
to compression 256*256
So far, after trained complete how can I generate my customized pre-training(airplane2Eva.pb) to let the module transfer all the image what I want
Or I need to follow this command below
python test.py --dataroot datasets/horse2zebra/testA --name horse2zebra_pretrained --model test
--name can change my dataset and names what I want?
And I also follow the repo below, but the result is not very well, so I want to compare what the difference between
https://github.com/vanhuyz/CycleGAN-TensorFlow
By the way, for the dataset, the airplane position will be the key to determine the result? Or what should I notice?

**Comment by junyanz** [comment]
Yes, you can use `test.py`. Please see the [instruction](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#cyclegan-traintest) 
It might be challenging to translate a CAD model to an entire image directly as it's hard to synthesize a background from a white background. You might need to segment the airplane and only run CycleGAN between segmented CAD model and segmented real images.

**Comment by junyanz** [comment]
CycleGAN assumes that images in different domain share similar content but with different styles. The white background has zero information and does not share content with the target domain.

**Comment by sudharavali** [comment]
@sigmagod Hi, how did you deal with this issue ? Did changing the backgrounds help you in the case ?

**Comment by sigmagod** [comment]
> @sigmagod Hi, how did you deal with this issue ? Did changing the backgrounds help you in the case ?

@sudharavali Finally, I use style transfer and texturegan to rebuild CAD model texture, then synthesis with background.
